Airwallex/Card/Internal/AWXCardViewController.swift

```diff
@@ -0,0 +1,669 @@
+        ])
+
+        container.addArrangedSubview(titleStack)
+        titleStack.addArrangedSubview(titleLabel)
+        titleStack.addArrangedSubview(closeButton)
+        [cardNoField, nameField, cvcStackView].forEach {
+            container.addArrangedSubview($0)
+        }
+        cvcStackView.addArrangedSubview(expiresField)
+        cvcStackView.addArrangedSubview(cvcField)
+
+        expiresField.widthAnchor.constraint(equalTo: cvcField.widthAnchor, multiplier: 1.7).isActive =
+            true
+
+        saveCardSwitchContainer.addArrangedSubview(saveCardLabel)
+        saveCardSwitchContainer.addArrangedSubview(saveCardSwitch)
+        if viewModel?.isCardSavingEnabled == true {
+            container.addArrangedSubview(saveCardSwitchContainer)
+        }
+        if viewModel?.isBillingInformationRequired == true {
+            container.addArrangedSubview(billingStackView)
+            billingStackView.addArrangedSubview(addressSwitchContainer)
+            addressSwitchContainer.addArrangedSubview(addressLabel)
+            addressSwitchContainer.addArrangedSubview(addressSwitch)
+            [firstNameField, lastNameField, countryView, stateField, cityField, streetField, zipCodeField, emailField, phoneNumberField].forEach {
+                billingStackView.addArrangedSubview($0)
+            }
+            firstNameField.next = lastNameField
+
+            phoneNumberField.next = stateField
+            stateField.next = cityField
+            cityField.next = streetField
+            streetField.next = zipCodeField
+            zipCodeField.next = emailField
+            emailField.next = phoneNumberField
+        }
+
+        container.addArrangedSubview(confirmButton)
+
+        cardNoField.next = nameField
+        nameField.next = expiresField
+        expiresField.next = cvcField
+
+        if let billing = viewModel?.initialBilling {
+            firstNameField.setText(billing.firstName ?? "", animated: false)
+            lastNameField.setText(billing.lastName ?? "", animated: false)
+            emailField.setText(billing.email ?? "", animated: false)
+            phoneNumberField.setText(billing.phoneNumber ?? "", animated: false)
+
+            let address = billing.address
+            if address != nil {
+                countryView.setText(viewModel?.selectedCountry?.countryName ?? "", animated: false)
+                stateField.setText(address?.state ?? "", animated: false)
+                cityField.setText(address?.city ?? "", animated: false)
+                streetField.setText(address?.street ?? "", animated: false)
+                zipCodeField.setText(address?.postcode ?? "", animated: false)
+            }
+        }
+        setBillingInputHidden(isHidden: viewModel?.isReusingShippingAsBillingInformation == true)
+        addressSwitch.isOn = viewModel?.isReusingShippingAsBillingInformation == true
+        saveCard = false
+    }
+
+    func addUnionPayWarningViewIfNecessary() {
```

Comment:
All these methods should be private.

Reply:
most of them require public because of @objc